### ISSUE TITLE
Review all commits

### ISSUE THREAD

**Issue by mrwonko** [mrwonko/OpenJK#84]
OpenJK aims to only fix issues, not change the gameplay. But reportedly, some changes have crept in. Still, it provides too many useful changes to completely disregard. Instead, I should help bring it into a shape that I'm happy to fork.

Sadly, I fear this involves reviewing every change in it. That sounds like a lot of work. But streaming the process could prove educational...

**Comment by xiaoxiao921** [comment]
Is there any known examples of gameplay changes?

**Comment by mrwonko** [comment]
Yeah, last year we found an accidental regression that went unnoticed for 16 years: JACoders/OpenJK#1127
